selfdrive/modeld/models/nav.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "cereal/messaging/messaging.h"
+#include "cereal/visionipc/visionipc_client.h"
+#include "common/util.h"
+#include "common/modeldata.h"
+#include "selfdrive/modeld/models/commonmodel.h"
+#include "selfdrive/modeld/runners/run.h"
+
+#define NAV // TODO: This should probably be defined in the sconscript or something
```

Comment:
will we ever want a build without this defined?

Reply:
Removed for now, this flag really belongs in driving.h with the other model input gates